samples/server/petstore/spring-mvc-j8-async/src/main/java/io/swagger/api/StoreApi.java

```diff
@@ -97,7 +97,7 @@
     @RequestMapping(value = "/store/order/{order_id}",
         produces = { "application/xml", "application/json" }, 
         method = RequestMethod.GET)
-    default CompletableFuture<ResponseEntity<Order>> getOrderById(@Min(1) @Max(5) @ApiParam(value = "ID of pet that needs to be fetched",required=true) @PathVariable("order_id") Long orderId) {
```

Comment:
Wasn’t this tested and passing on CI ?

Reply:
I ran the shell script under ./bin/ as it is mentioned in the checklist and it re-generated java files. I understood it as something I have to do in order to help CI build/tests. 
If I am wrong, I'll remove them.

Reply:
Forget my comment. It's passing because `1`and `5`are small numbers so they don't need the `L` suffix